syntax = "proto3";
package Protobuf;
option csharp_namespace = "Communication.Proto";

message PingPacket { int64 Ticks = 1; }
